src/adapters/claude/helpers/completions.ts

```diff
@@ -42,13 +43,7 @@ export class AnthropicCompletion extends SuperAnthropic {
     return tokenLimits.get(model) || 4096;
```

Comment:
You should also throw the same error instead of defaulting to 4096 for every model randomly.